community.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Community Resources
+layout: single
+---
+
+We are experimenting with a variety of communication channels:
+
+* **General feedback and chat** is welcomed in both[^both] [Slack][] and [Discord][]. We tend to 
+use Slack for real-time chat between contributors, and Discord for casual
+  conversation and the occasional video huddle, but feel free to use either.
+* **Technical questions** should be directed to [GitHub Discussions][discuss], where the 
+information can be discovered through web searches. Let's help make the next person's Google search a
+  little easier!
+* **Bug reports and feature requests** should go into [GitHub issues][new-issue] so we can
+  prioritize and track the work.
+* **Announcements** will be posted to our [News][] page here, and to Discussions, Slack, 
+and Discord. (Unfortunately, only the chat services support push notifications.)
+
+All of the above is subject to change as we see where people gravitate.
+
+⚠️ This product is not owned or supported by Amazon or AWS, but by a small group of volunteers.
+At this time we cannot promise any particular SLA for responding to issues.
+{: .notice--warning}
+
+[^both]: Each app has unpleasant tradeoffs, and it's unclear which will work best.
```

Comment:
Minor: given that we'd ideally like to settle on a single one, is it worth framing this that way? Something like:
```suggestion
[^both]: Each app has unpleasant tradeoffs, and we hope the community's experience will show what works best. Feedback is welcome!
```

(I realize this is implied by the "subject to change" language on line 19, but just a thought to take or leave)